OracleDatabase/SingleInstance/dockerfiles/19.3.0/checkDBStatus.sh

```diff
@@ -18,10 +18,10 @@ ORAENV_ASK=NO
 source oraenv
 
 # Check Oracle at least one PDB has open_mode "READ WRITE" and store it in status
-status=`sqlplus -s / as sysdba << EOF
+status=`sqlplus -s / << EOF
    set heading off;
    set pagesize 0;
-   SELECT DISTINCT open_mode FROM v\\$pdbs WHERE open_mode = '$OPEN_MODE';
+   SELECT DISTINCT open_mode FROM v\\$database WHERE open_mode = '$OPEN_MODE';
```

Comment:
I did that because for oracle user (externally mapped from system) the `v$pdbs` query returned no rows. But having read some [blog](http://oracledbpro.blogspot.com/2015/09/cant-view-data-via-common-user-in.html) I think we are good with querying `v$pdbs` again.